atlasdb-impl-shared/src/main/java/com/palantir/atlasdb/debug/LocalLockTracker.java

```diff
@@ -94,6 +97,15 @@ void logRefreshResponse(Set<ConjureLockToken> tokens, ConjureRefreshLocksRespons
         eventBuffer.add(event);
     }
 
+    public void logRefreshV2Response(Set<ConjureLockTokenV2> tokens, ConjureRefreshLocksResponseV2 response) {
```

Comment:
This class is deprecated, and while removing that whole piece is not in the scope of this PR, surely we can just not do this? In any case, not checking correctness here

Reply:
Fair enough. Yeah I won't bother then - getting a correct impl wasn't too hard, but it's fair to say we aren't actively supporting this